content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,122 @@
+
+**Spring Boot**
+
+Spring Boot is an open source Java-based framework used to create a micro Service. Many Real world applications are written in Spring Boot for faster development and better maintainability. More information can be found at** <https://spring.io/projects/spring-boot>**
+
+Note: We can move the Spring Boot application to Quarkus with no change in code.  Moving code to Quarkus will reduce the footprint of Spring Boot application. More information can be found at <https://quarkus.io/blog/quarkus-for-spring-developers/>
+
+In this blog, we will checkout Spring Boot application from Git , customise it and execute. 
+
+**Application Prerequisites**
+
+<!--\\\[if !supportLists]-->a.      <!--\\\[endif]-->K3S cluster must be accessible. Note down the control plan node details. This information is required to deploy the Spring Boot application
+
+<!--\\\[if !supportLists]-->      <!--\\\[endif]-->
+
+b.      Access the HPE Data Fabric Object Store service UI running on port 9000. For example URL  - https://FQDN:9000/ .  Note down the access key and secret key. Default access key and secret key is “minioadmin” and “minioadmin” respectively. It is advised to change the default values.
+
+<!--\\\[if !supportLists]-->c.      <!--\\\[endif]-->Java 11 , Apache Maven 3.8+ , Docker Client 
+
+**Build and Install Steps**
+
+<!--\\\[if !supportLists]-->1.    <!--\\\[endif]-->Check out the code from git <https://github.hpe.com/kiran-mavatoor/df-s3-springboot-k3s-demo>  and go the project directory
+
+<!--\\\[if !supportLists]-->
+
+2.    <!--\\\[endif]-->Copy the ssl_usertruststore.p12 from the HPE Data Fabric cluster into certs folder under project directory. The ssl_usertruststore.p12 file is located at /opt/mapr/conf directory in cluster node. The password for p12 can be copied from “ssl.client.truststore.password” property value in /opt/mapr/conf/ssl-client.xml
+
+<!--\\\[if !supportLists]-->
+
+3.    <!--\\\[endif]-->From project directory, open resources/application.properties. Change the key values as per your environment. 
+
+<!--\\\[if !supportLists]-->
+
+4.    <!--\\\[endif]-->Execute “mvn clean install”
+
+<!--\\\[if !supportLists]-->
+
+5.    <!--\\\[endif]-->The distributable is available in target/df-s3-springboot-k3s-demo-1.0-SNAPSHOT.jar
+
+<!--\\\[if !supportLists]-->6.    <!--\\\[endif]-->Edit the DockerFile located in project directory. The value of 
+
+“-Djavax.net.ssl.trustStorePassword”  must be same as the value of  “ssl.client.truststore.password” obtained from step 2. 
+
+Note:- This value can be configured using config-map.yaml in K3S cluster.
+
+<!--\\\[if !supportLists]-->7.    <!--\\\[endif]-->Execute below docker commands to build docker image and push it to docker hub. 
+
+Note:- We can use podman instead of docker to create images. More information on podman can be obtained from <https://developers.redhat.com/blog/2020/11/19/transitioning-from-docker-to-podman#transition_to_the_podman_cli>
```

Comment:
You can use markdown syntax for Note >. And remove the - after the :
Like for Quarkus, is it important to discuss podman here, or is this just adding confusion.